.cargo/refresh-tables.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
Uh, I forgot to mention that I renamed `/.cargo/` to `/etc/`, since it now has non-cargo scripts in it. We should move this to the new dir, as well.